generator/osm_element.cpp

```diff
@@ -121,6 +121,20 @@ string OsmElement::ToString(string const & shift) const
   return ss.str();
 }
 
+bool OsmElement::GetTag(string const & key, string & value) const
```

Comment:
Cause it would be impossible to distinguish between empty tag an the absence of one.